### PR TITLE
Add named exports option

### PR DESCRIPTION
**Pull request by timse**
Add option to generate named exports instead of an interface.

TODO: 
- [x] add `named exports` generator
- [x] warn if classes would go missing due to filter for dashed-names
- [x] document option in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +79,7 @@ webpackConfig.module.loaders: [
 ## Example
 
 Imagine you have a file `~/my-project/src/component/MyComponent/component.scss` in your project with the following content:
```
Comment (KingHenne) [inline README.md]: Should be `mycomponent.scss`.

File: README.md [hunk 2]
```diff
@@ -54,6 +103,14 @@ declare const styles: IMyComponentScss;
 export default styles;
 ```
 
+### using `namedExport`
+Using the `namedExport`-option the generated file will look as follow:
```
Comment (KingHenne) [inline README.md]: "Using the `namedExport` as well as the `camelCase` options ..."

File: src/index.js [hunk 3]
```diff
@@ -21,18 +25,31 @@ module.exports = function(input) {
...
-    const cssModuleInterfaceFilename = filenameToTypingsFilename(requestedResource);
-    const cssModuleInterface = generateInterface(cssmodules, requestedResource);
-    persist.writeToFileIfChanged(cssModuleInterfaceFilename, cssModuleInterface);
+    let cssModuleDefinition;
+    if (!query.namedExport) {
+      cssModuleDefinition = generateGenericExportInterface(cssModuleKeys, filename);
+    } else {
+      const [cleanedDefinitions, skippedDefinitions] = filterNonWordClasses(cssModuleKeys);
+      if (skippedDefinitions.length > 0 && !query.camelCase) {
+        console.warn(`Typings for CSS-Modules: option 'namedExport' was set but 'camelCase' for the css-loader not.
```
Comment (KingHenne) [inline src/index.js]: Helpful! 👍

File: README.md [hunk 4]
```diff
@@ -103,8 +103,8 @@ declare const styles: IMyComponentScss;
 export default styles;
 ```
 
-### using `namedExport`
-Using the `namedExport`-option the generated file will look as follow:
+### using `namedExport` with the `camelCase`-option
+Using the namedExport as well as the camelCase options the generated file will look as follow:
```
Comment (KingHenne) [inline README.md]: by copy pasting, the back ticks got lost

Comment (timse) [inline README.md]: oh nooo i will amend

**Pull request merged**